### ISSUE TITLE
Sparkler Elasticsearch storage engine

### ISSUE THREAD

**Issue by lewismc** [USCDataScience/sparkler#209]
We are moving ahead with this project within the USC CSCI 401 Senior Capstone Program. My stakeholder needs are as follows
```
1. As a Crawler Administrator, I need to be able to specify and configure the Sparkler storage engine through configuration parameters.
1. As a Data Analyst, I need access to a dashboard which displays crawl information.
1. As a DevOps Engineer, I need to deploy Sparkler (configured with the Elasticsearch storage engine) via Docker and as a Helm Chart.
1. As a Test and Quality Assurance Engineer, I need to integrate Sparkler tests for the Elasticsearch storage engine into my CI process.
1. As a Development Lead, I need access to developer documentation covering the Elasticsearch storage engine for Sparkler.
```
**Sparkler Committers**, I wonder if it is possible for us to use the [Github projects](https://github.com/USCDataScience/sparkler/projects) feature to manage the project?

**Capstone Team**, please reply with your Github ID here.

**Comment by Kefaun2601** [comment]
Github ID: 42557579

**Comment by slhsxcmy** [comment]
login: slhsxcmy
id: 20136533

**Comment by nhandyal** [comment]
uid: nhandyal

**Comment by felixloesing** [comment]
My Github ID is felixloesing

**Comment by KilometersFan** [comment]
ID: 35278719, Username: KilometersFan

**Comment by thammegowda** [comment]
> **Sparkler Committers** , I wonder if it is possible for us to use the Github projects feature to manage the project?

Yes. Of course. 
@lewismc Let me know what permissions etc you need. FYI I already sent an invitation to your account.

**Comment by lewismc** [comment]
@thammegowda please add all folks mentioned in this thread as contributors. They will need to use the [Project's function](https://github.com/USCDataScience/sparkler/projects/1) to manage the ongoing project.

**Comment by lewismc** [comment]
@thammegowda once don, please close this issue. THANK YOU very much @thammegowda

**Comment by thammegowda** [comment]
1. I created a team @USCDataScience/csci401-2021spring
2. Added all the above 5 members to that team
3. Everybody in the team is granted Truage permission which allows them to manage issues and pull requests.  
4. I think their best course of action is 1. clone this repo to their own space, 2. modify, 3. send pull request so we can review and merge. We shall give write permissions over time as they make progress and prove to be fit (similar to the ASF ).
